### ISSUE TITLE
NVDA says 'grouping' when go from grid header to grid itself, but no grouping applied

### ISSUE THREAD

**Issue by VolhaBertash** [angular-ui/ui-grid#6327]
NVDA says 'grouping' when go from grid header to grid itself, but no grouping is applied. If proceed navigation to another cell, NVDA doesn't repeat 'grouping' anymore. If any cell is selected and during navigation focus goes not to first cell, but to selected one, issue is also reproduced.

STEPS TO REPRODUCE:
1. Turn on NVDA.
2. Go to page with grid, e.g. http://ui-grid.info/docs/#/tutorial/122_accessibility.
3. Move focus till last field in the filter row.
4. Press TAB. -> Focus goes to the whole table and NVDA reads out 'table editable grouping'.
5. Press TAB. -> Focus goes to the first cell in the grid and NVDA reads out 'Ethel Price grouping'.
Expected result: NVDA doesn't read out 'grouping' as it's not applied and can confuse accessible user. In it should be consistent - if grouping applied, but NVDA reads out it only for first cell, so no 'grouping' should be read out after navigation from the grid header to cell in the grid.

I am adding attachments with reproducing. There I:
1. set focus to input field in gender column.
2. pressed TAB -> NVDA reads out 'grouping' for whole grid.
3. pressed TAB -> NVDA reads out 'grouping' for first cell.
4. pressed TAB -> NVDA doesn't read out 'grouping' for second cell.

In the bug description I mainly focused on the reading out 'grouping' for the first cell as exactly this issue affects our application, but reading 'grouping' for whole grid also doesn't look like expected, so please, pay attention on this as well.

[NVDA Grid Grouping.zip](https://github.com/angular-ui/ui-grid/files/1161717/NVDA.Grid.Grouping.zip)

**Comment by orneryd** [comment]
Thanks for contributing! This repository has been transferred, and we're preparing the new 5.x release. We're closing older issues and pull requests as part of the reset. Please join us on Discord to follow the work or continue the conversation: https://discord.gg/Baz4w8ZWN
